logic/auth.js

```diff
@@ -40,6 +46,33 @@ function getCachedPassword() {
     return devicePassword;
 }
 
+// Change the device name
+async function changeName(name) {
+    resetChangeNameStatus();
+
+    changeNameStatus.percent = 1; // eslint-disable-line no-magic-numbers
+
+    changeNameStatus.percent = 40; // eslint-disable-line no-magic-numbers
+
+    try {
+        // get user data
+        const user = await diskLogic.readUserFile();
+
+        // update user name
+        user.name = name;
```

Comment:
It won't change a lot of things as it's almost instant, but it seems more logical to me to have the second step (40%)  after the file read instead of just after the first step (1%).
```suggestion
    changeNameStatus.percent = 1; // eslint-disable-line no-magic-numbers

    try {
        // get user data
        const user = await diskLogic.readUserFile();
        
        changeNameStatus.percent = 40; // eslint-disable-line no-magic-numbers
        
        // update user name
        user.name = name;
```

Reply:
Yeah I guess it's near to instant. In fact just before the try it set 40% even if the try succeed, with your change if it fails we do 1 to 100.
What do you think?

Reply:
Maybe 1-30-60-100 🧐

It's not really important haha